examples/processor/pyproject.toml

```diff
@@ -0,0 +1,22 @@
+[tool.poetry]
+name = "processor"
+version = "0.1.0"
+description = ""
+authors = ["Reid Meyer <reid.meyer@kpn.com>"]
+
+[tool.poetry.dependencies]
+python = "^3.10"
+aiorun = "^2022.4.1"
+kstreams = { path = "../../.", develop = true }
+asyncio = "^3.4.3"
+uvicorn = "^0.18.2"
```

Comment:
Where is uvicorn used?

Reply:
good catch. I was using it at first, but then removed. I will remove.